website/docs/r/bot_channel_directline.html.markdown

```diff
@@ -58,14 +58,24 @@ A `site` block has the following properties:
 
 * `name` - (Required) The name of the site
 
+* `application_id` - (Optional) The application ID for this site.
+
+* `block_user_upload_enabled` - (Optional) Is the block user upload enabled for this site? Defaults to `false`.
+
 * `enabled` - (Optional) Enables/Disables this site. Enabled by default Defaults to `true`.
 
+* `endpoint_parameters_enabled` - (Optional) Is the endpoint parameters enabled for this site?
+
+* `no_storage_enabled` - (Optional) Is no storage enabled for this site?
```

Comment:
I've checked the meaning of this property in the API, it says:

> Whether this no-storage site is disabled detailed logging for?

This doesn't make a lot of sense to me either. Could you please clarify what does this property actually do? Since we shall give it a better name to avoid any potential confusion..

Reply:
Service team also cannot provide more description about it.

Reply:
Probably we shall also rename the property itself, to something like `no_detailed_logging_enabled`?

Reply:
I suggest to keep align with api spec. I prefer to improve the property description. Does it make sense?